packages/remix-react/transition.ts

```diff
@@ -81,29 +75,6 @@ export type TransitionStates = {
 
 export type Transition = TransitionStates[keyof TransitionStates];
 
-export type Redirects = {
-  Loader: {
-    isRedirect: true;
-    type: "loader";
-    setCookie: boolean;
-  };
-  Action: {
-    isRedirect: true;
-    type: "action";
-    setCookie: boolean;
-  };
-  LoaderSubmission: {
-    isRedirect: true;
-    type: "loaderSubmission";
-    setCookie: boolean;
-  };
-  FetchAction: {
-    isRedirect: true;
-    type: "fetchAction";
-    setCookie: boolean;
-  };
-};
-
```

Comment:
Removed a few thigs here that were exported but never used elsewhere - leftovers from TM implementation